### ISSUE TITLE
"Single page" view settings too hidden

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#505]
```
What steps will reproduce the problem?
1. Be a noob
2. Download ebookdroid, and read in the description "We do have page turning 
emulation - just switch to single page mode and choose page turning effect of 
your choice!"
3. Try to find "single page mode"

What is the expected output?
This should be very easy to find.
I would expect it to be under the "magnifying glass (?)" icon, that has "full 
screen" "Show title", etc.

What do you see instead?
This setting is hidden very deeply.
All Setting - makes sense, though a direct view mode menu would be better
Rendering  - not really the place I'd expect this setting. Rendering sounds 
more like issues about fonts and stuff.
Document View mode - makes sense. But it is the 5th option down, so one has to 
read fairly carefully.

It is also available in Book settings, and there is a bit less hidden. But the 
noob (me) is looking to have ALL books behave in a certain way


What version of the product are you using?
1.6.6

On what operating system?
Android 4.1.2

What device are you using?
Galaxy Note 2

Please provide any additional information below.
```

Original issue reported on code.google.com by `mlachm...@gmail.com` on 14 May 2013 at 4:40

**Comment by GoogleCodeExporter** [comment]
```
I noticed this is fixed in 2.0.0-beta-r2619, where the magnifier menu has a 
"Document view mode" entry.
Maybe change the text in the store to:
"We do have page turning emulation - just switch to single page document view 
mode and choose page turning effect of your choice!"

Thanks!
```

Original comment by `mlachm...@gmail.com` on 14 May 2013 at 7:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 12:38
- Changed state: **Fixed**
